dots_and_boxes/__tests__/gameLogic_test.ts

```diff
@@ -0,0 +1,74 @@
+import {IMove} from '../../common/common';
```

Comment:
All needed here is to delete the specifics imports, if you are going to use them in the future you can just comment them right now.